supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -117,22 +117,31 @@ object Run {
   }
 
   def interpretOperationStatus(op: Operation): RunStatus = {
-    if (op.getDone) {
-      lazy val eventList = getEventList(op)
-      lazy val ceInfo = op.getMetadata.get ("runtimeMetadata").asInstanceOf[GArrayMap[String,Object]].get("computeEngine").asInstanceOf[GArrayMap[String, String]]
-      lazy val machineType = Option(ceInfo.get("machineType"))
-      lazy val instanceName = Option(ceInfo.get("instanceName"))
-      lazy val zone = Option(ceInfo.get("zone"))
-
-      // If there's an error, generate a Failed status. Otherwise, we were successful!
-      Option(op.getError) match {
-        case None => Success(eventList, machineType, zone, instanceName)
-        case Some(error) => Failed(error.getCode, Option(error.getMessage), eventList, machineType, zone, instanceName)
+    try {
+      if (op.getDone) {
+        lazy val eventList = getEventList(op)
+        lazy val computeEngineOption = for {
+          runtimeMetadata <- op.getMetadata.asScala.get("runtimeMetadata")
+          computeEngine <- runtimeMetadata.asInstanceOf[GArrayMap[String, Object]].asScala.get("computeEngine")
+        } yield computeEngine.asInstanceOf[GArrayMap[String, String]].asScala
```

Comment:
ToL: I'm 99% sure I'd prefer these to be a placeholder like "Undeterminable" in the metadata, rather than failing a job just because we couldn't find the machine type... PO decision?